app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/drone/ConnectedDroneActivityTest.kt

```diff
@@ -34,7 +34,7 @@ import org.mockito.Mockito.verify
 @UninstallModules(DroneModule::class)
 class ConnectedDroneActivityTest {
 
-    val activityRule = ActivityScenarioRule(ConnectedDroneActivity::class.java)
+    private val activityRule = ActivityScenarioRule(ConnectedDroneActivity::class.java)
```

Comment:
Why this one is private and not the other?

Reply:
It's not that one is private and not the other, it's that it was useless to keep this variable public (used only in this test), so I changed it